public/build/asset-manifest.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
build folder should be added to `.gitignore`. It shoud not be commited to githhub